src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/RSA/ImportExport.cs

```diff
@@ -13,6 +13,11 @@ public partial class ImportExport
         [Fact]
         public static void ExportAutoKey()
         {
+            if (PlatformDetection.IsSmallerThan462)
```

Comment:
Actually we can do that as soon that should be fixed and it is a good thought :)